locust/main.py

```diff
@@ -8,6 +8,7 @@
 import time
 
 import gevent
+from flask_basicauth import BasicAuth
```

Comment:
This import doesn't seem to be used.

Reply:
> 
> 
> LGTM, and I do see the use case. The only drawback is that the list of arguments for locust is getting pretty long.
> 
> Could you find somewhere to add it in the documentation?

I agree but maybe a separate PR though, what do you think ?